### ISSUE TITLE
How do I check workflow?

### ISSUE THREAD

**Issue by sharmistharanit** [ubsuny/23-Homework4G5#43]
If I created a workflow, How can I access that? If I want to make any changes, can I do that?

**Comment by tirthbha** [comment]
@sharmistharanit yes you can do changes as you desired, even you can create workflow manually.

**Comment by sharmistharanit** [comment]
How do I create workflow manually? Can you please explain that? @tirthbha

**Comment by reshnashrestha** [comment]
You have to design your workflow for the specific task. You can create wrork flow from actions.
![image](https://github.com/ubsuny/23-Homework4G5/assets/143649458/27d768a9-aa24-4a36-b156-a11975c14b5e)

**Comment by s4il3sh** [comment]
Also, you can edit files inside the workflows directory to create tasks.
